src/html.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function HTML(props) {
+  return (
+    <html {...props.htmlAttributes}>
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="x-ua-compatible" content="ie=edge" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        {props.headComponents}
+      </head>
+      <body {...props.bodyAttributes}>
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `
+            window.fbAsyncInit = function() {
```

Comment:
To be honest I don't really like to have it on every page. If I understand it correctly, now it fetches data in browser and not at build time. Can we make it more like instagram photos that are downloaded at build time?

Reply:
Tried putting it inside `akcie.jsx` but it didn't work, yes I also don't like this and I'll move it in future. By fetching events through build script, we'll have to rebuild the site every time we create new event or update existing one, right?

Reply:
> By fetching events through build script, we'll have to rebuild the site every time we create new event or update existing one, right?

Yeah, you are right, I forgot. For Instagram we have set Zapier, that listens for new posts and triggers new build through web hook. Hopefully the same could be set for Facebook.

If not, then we have a problem :)

Previously, before Zapier, we had daily cron that run a new build once a day. It worked, but it was not the best solution either.

Reply:
@crazko can you pls add Zapier credentials into 1password? Thank you!

Reply:
Actually, Zapier is handled by @Ljuk67, so we need to ask him :)